log4j-core/src/main/java/org/apache/logging/log4j/core/async/AsyncLoggerDisruptor.java

```diff
@@ -122,7 +122,7 @@ public Thread newThread(final Runnable r) {
         final ExceptionHandler<RingBufferLogEvent> errorHandler = DisruptorUtil.getAsyncLoggerExceptionHandler();
         disruptor.setDefaultExceptionHandler(errorHandler);
 
-        final RingBufferLogEventHandler[] handlers = {new RingBufferLogEventHandler()};
+        final RingBufferLogEventHandler4[] handlers = {RingBufferLogEventHandler4.create()};
```

Comment:
I would rather point to `RBLEH.create()` I suggested below.

Reply:
Done: moved to a `createEventHandler` method in this class.